__tests__/unit/client/client.test.ts

```diff
@@ -236,30 +359,20 @@ describe("Client", () => {
       (axios.get as jest.Mock).mockResolvedValue(res);
 
       // Act
-      const result = await client.getPendingDeposits(zkpKeys);
+      const result = await client.getPendingDeposits(
+        zkpKeys,
+        tokenContractAddresses,
+      );
 
       // Assert
       expect(axios.get).toHaveBeenCalledWith(url, {
         params: {
           compressedZkpPublicKey: zkpKeys.compressedZkpPublicKey,
+          ercList: tokenContractAddresses,
         },
       });
       expect(result).toBe(tokenBalances);
     });
-
-    test("Should return null if client app responds with status outside the successful range", async () => {
-      // Arrange
-      (axios.get as jest.Mock).mockRejectedValue(
-        new Error("Axios error at commitment/pending-deposit"),
-      );
-
-      // Act
-      const result = await client.getPendingDeposits(zkpKeys);
-
-      // Assert
-      expect(axios.get).toHaveBeenCalledTimes(1);
-      expect(result).toBeNull();
-    });
   });
```

Comment:
So beautiful to see that SDK is becoming strong with all these tests. Hopefully, we can see now the value of Unit Tests!